app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
```

Comment:
But actually, there's quite a big reason for ENV, which is testability. We need to be able to deploy to demo (or somewhere) and switch it on now.